lib/improver/tests/blending/weighted_blend/test_rationalise_blend_time_coords.py

```diff
@@ -41,6 +41,7 @@
 
 from improver.utilities.cube_manipulation import merge_cubes
 from improver.blending.weighted_blend import rationalise_blend_time_coords
+from improver.tests.set_up_test_cubes import set_up_variable_cube
 
```

Comment:
When I run these unit tests I'm getting the following Warning messages. They might of been in the original but it would be nice to get rid of them

improver/utilities/cube_manipulation.py:361: UserWarning: Do not know what to do with mosg__model_configuration will delete it - value is uk_det
  warnings.warn(msg)
improver/utilities/cube_manipulation.py:361: UserWarning: Do not know what to do with mosg__model_configuration will delete it - value is uk_ens

Reply:
I can suppress this warning - it's arising for a cycle blending unit test where the warning isn't relevant.